tests/test_system_verilog_target.py

```diff
@@ -187,7 +187,6 @@ class _Foo(m.Circuit):
     tester.compile_and_run(
         target="system-verilog",
         simulator="vcs",
-        skip_compile=True,
```

Comment:
@leonardt wait why did we get rid of this? The idea was to just run the testbench against existing verilog, to isolate from magma -> verilog compilation since we only want to test that the testbench is generated correctly.

Reply:
It looks like this won't run because it needs MLIR so instead I'll remove the skip_run flag then.